rdkit_valgrind.suppressions

```diff
@@ -0,0 +1,3235 @@
+{
```

Comment:
Is there a good place to document the requirements for meaningful valgrind?

My understanding is that it only makes sense in a debug build, and that valgind and gcc versions need to match the ones used in the suppressions file. I suppose at minimum, the git commit message should include those details. Possibly also a comment at the top of this file?

Reply:
Yes, I will add those.